### ISSUE TITLE
Добавить словам с пол разговорные варианты в парадигму

### ISSUE THREAD

**Issue by svbichineva** [OpenCorpora/opencorpora#143]
Сейчас слово "полметра" имеет такие формы: нет полуметра, дать полуметру, любоваться полуметром, говорить о полуметре
Добавить: полметра (род), полметре (дат), полметром (твор), полметре (предл), т.е. без "у" посередине 

Другие слова:
Полкилометра
Полсантиметра

**Comment by svbichineva** [comment]
Может, ты как-нибудь сможешь найти все слова с началом "пол"?

**Comment by grandsbor** [comment]
Конечно, могу, но вряд ли тебя интересуют слова типа "полковник", правда? :) Какой критерий выбора?

**Comment by svbichineva** [comment]
А, видимо, начинается на пол и заканчивается на "ы/и/а" (хотя половина тоже попадет, но я отсмотрю глазами)

**Comment by grandsbor** [comment]
[316 штук](https://yadi.sk/i/46hG2Nn5eA2DY)

**Comment by svbichineva** [comment]
А если я перечислю здесь слова, ты сможешь автоматически добавить 4 формы в парадигму каждого слова, по модели
Полчаса, sing, gent, Infr
Полчаса, sing, datv, Infr,
Полчаса, sing, ablt, Infr
Полчаса, sing, loct, Infr
Или мне вручную сделать?

**Comment by grandsbor** [comment]
Если их не больше 20, быстрее руками

**Comment by svbichineva** [comment]
Дим, найди еще, пожалуйста, слова, начинающиеся с "полу"

**Comment by grandsbor** [comment]
[90 штук](https://yadi.sk/i/x7uSef7IeaUxY)
